### PR TITLE
Make organization dropdown scrollable when using mouse wheel

### LINKED ISSUE #5580: Organization dropdown not scrollable when using mouse wheel
- Gitea version (or commit ref): b77c3eb
- Operating system: 
    - Host: Ubuntu 18.04 4.15.0-33-generic
    - Container: gitea/gitea:latest
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (Dashboard tab https://try.gitea.io/)
  - [ ] No
  - [ ] Not relevant

## Description

When scrolling the mouse wheel while hovering over the organization dropdown the whole page is scrolled instead of just the elements inside the dropdown. This behaviour feels weird and it's inconsistent with other dropdowns such as the "Owner" dropdown when creating a new repository.

## Screenshots

dropdown

### PR DESCRIPTION
**Pull request by metiftikci**
Old:
ddlold

New:
ddlnew

Fix #5580

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: templates/user/dashboard/navbar.tmpl [hunk 1]
```diff
@@ -11,7 +11,7 @@
 					<div class="ui header">
 						{{.i18n.Tr "home.switch_dashboard_context"}}
 					</div>
-					<div class="items">
+					<div class="scrolling menu" style="border-radius: 0px !important;box-shadow: none !important;border-bottom: 1px solid rgba(34, 36, 38, 0.15);">
```
Comment (lafriks) [inline templates/user/dashboard/navbar.tmpl]: Inline style should be moved to less/CSS class

**Review by lafriks — CHANGES_REQUESTED** [review]

File: templates/user/dashboard/navbar.tmpl (hunk 1 above)
Comment (zeripath) [reply templates/user/dashboard/navbar.tmpl]: Sorry I missed that!

Comment (techknowlogick) [reply templates/user/dashboard/navbar.tmpl]: @zeripath no need to be sorry. This is why there is a policy to have at least two acceptance as sometimes one person may miss something that another doesn't (and vice versa).

Comment (metiftikci) [reply templates/user/dashboard/navbar.tmpl]: done

Comment (metiftikci) [reply templates/user/dashboard/navbar.tmpl]: @lafriks can you check please sir

File: public/less/_base.less [hunk 2]
```diff
@@ -386,33 +386,12 @@ pre > code {
...
-            -webkit-touch-callout: none;
-            &.active {
-                font-weight: 700;
-            }
-            &:hover {
-                background: rgba(0,0,0,.05);
-                color: rgba(0,0,0,.8);
-                z-index: 13;
-            }
+.ui.floating.dropdown {
```
Comment (lafriks) [inline public/less/_base.less:399]: why remove so much css styles?

Comment (metiftikci) [reply public/less/_base.less:399]: Its old styles for dropdown called 'items' class.
they wont be need when use semantic.ui original classes 'scrolling menu'

**Comment by lunny** [comment]
conflicted

**Comment by metiftikci** [comment]
@lunny conflict resolved

**Comment by metiftikci** [comment]
@lafriks can you review pls

**Review by lafriks — APPROVED** [review]

**Comment by techknowlogick** [comment]
@muhammedtiftikci thanks for PR. Now that this has been merged could you send a backport to the release/v1.7 branch? If you need assistance please feel free to ask, and we can help 😄

**Pull request merged**